src/components/tabs/Tabs.vue

```diff
@@ -15,6 +15,10 @@
   import {createUniqueId} from '../../utils'
 
   export default {
+    props: {
+      fullWidth: Boolean
```

Comment:
I'm for changing this to `vaFullWidth` as it's a custom extension of AUI's capabilities.